std_msgs/Header header
MaskData[] mask_data
sensor_msgs/Image rgb_img
sensor_msgs/Image depth_img